extension/src/NABfunctions.ts

```diff
@@ -501,16 +501,21 @@ export async function generateToolTipDocumentation(): Promise<void> {
         title: "Generating ToolTip Documentation...",
       },
       () => {
-        return new Promise<void>((resolve) => {
+        return new Promise<void>((resolve, reject) => {
           setTimeout(async () => {
             await ToolTipsDocumentation.generateToolTipDocumentation(
```

Comment:
Do we still need the `await` when using `.then`?

Reply:
Nice catch! Apparently we don't using `await` is equivalent to using `.then(() => ...)` if I understand [this article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) correctly.